packages/ui-react/src/components/TileDock/TileDock.tsx

```diff
@@ -263,13 +268,17 @@ function TileDock<T>({
             />
           ) : null}
           {tileList.map((tile: Tile<T>, listIndex) => {
+            const posInSet = items.findIndex((item) => item === tile.item); // TODO optimize this for performances
```

Comment:
I could not find a more performant way to determine the actual position of the item.
Because of virtualization and circular rotation, the indexes of the shelf items in a playlist does not correspond with the index in the DOM.

Reply:
Smart fix. I wonder about performance indeed, but let's evaluate that later. 
Perhaps a minor performance win could be made by comparing `item.mediaid === tile.item.mediaid`.